families/track_and_trade/client/src/views/list_fish.js

```diff
@@ -40,6 +40,9 @@ const FishList = {
 
   view (vnode) {
     let publicKey = api.getPublicKey()
+    vnode.state.filteredRecords.sort((a, b) => {
```

Comment:
`vnode.state.records` is the source of truth, and `filteredRecords` are just slices of that.  So you can probably get by just sorting once on load.